### DISCUSSION TITLE
Can ART show accuracy while using "classifier.fit()" on pytorch

### DISCUSSION THREAD

**Discussion by UnReAlKiNg** [Trusted-AI/adversarial-robustness-toolbox#1904] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by beat-buesser** [comment]
Hi @UnReAlKiNg No, not out of the box. You'll have to modify your model with callbacks or modify method `.fit()`.
